public/locales/hi/translation.json

```
@@ -117,6 +117,8 @@
   "delhi_shiksha_PPsCompletionMessage": "<div class='text-class'>अपना समय निकालकर अपनी राय देने और हमारे बच्चों के लिए बेहतर भविष्य बनाने में योगदान देने के लिए आपका बहुत-बहुत धन्यवाद।</div>",
   "odisha_youth_PPsCompletionMessage": "<div class='text-class'>साझा करने के लिए धन्यवाद – आपकी यात्रा महत्वपूर्ण है। आगे बढ़ते रहिए, आप सचमुच बदलाव ला रहे हैं।</div>",
   "fgd_teacher_PPsCompletionMessage": "<div class='text-class'>आपके समय के लिए धन्यवाद। बेसलाइन रिपोर्ट का विश्लेषण पूरा होने के बाद, हम आपको इससे संबंधित महत्वपूर्ण जानकारी साझा करेंगे।</div>",
+  "stakeholder_fgd_PPsCompletionMessage": "<div class='text-class'>धन्यवाद! आपका जवाब सफलतापूर्वक दर्ज कर दिया गया है।</div>",
```

Biome (2.4.14)

[error] The key stakeholder_fgd_PPsCompletionMessage was already declared.

(lint/suspicious/noDuplicateObjectKeys)